client/lib/search-box.js

```diff
@@ -12,7 +12,7 @@ module.exports = function () {
   });
   searchinput.addEventListener('keyup', debounce(function (e) {
     var q = e.target.value;
-    if (q.length > 0) {
+    if (q.length > 0 && e.code !== 'Enter') {
```

Comment:
Enter is the [correct value](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values), but maybe we should use [event.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) instead, as it has better support than [event.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)